src/functions/certGen.ts

```diff
@@ -1,57 +1,26 @@
-import { DeleteObjectCommandOutput, PutObjectCommandOutput } from '@aws-sdk/client-s3';
-import { Callback, Context, Handler, SQSEvent, SQSRecord } from 'aws-lambda';
+import { Callback, Context, Handler, SQSBatchItemFailure, SQSBatchResponse, SQSEvent, SQSRecord } from 'aws-lambda';
```

Comment:
Can you remove this unused import please

Reply:
Thanks :)